tests/conftest.py

```diff
@@ -603,3 +632,17 @@ def test_perf_alert(test_perf_signature, test_perf_alert_summary):
         prev_value=100.0,
         new_value=150.0,
         t_value=20.0)
+
+
+@pytest.fixture
+def test_perf_alert_onhold(test_perf_signature, test_perf_alert_summary_onhold):
```

Comment:
Could you put these fixtures in `test_performance_alertsummary_api.py` ? They seem very specific to this test.